lib/shims/eq/R2015b/isfolder.m

```diff
@@ -0,0 +1,9 @@
+function result = isfolder(folderName)
```

Comment:
Please use `out` instead of `result` as the name of a single function output argument.

Reply:
ok, I will change to honor this style preference, and get this into the open PR